CONTRIBUTING.md

```diff
@@ -1,19 +1,238 @@
-## Welcome!
-
-We're so glad you're thinking about contributing to an HHS IDEA Lab open source project! If you're unsure about anything, just ask -- or submit the issue or pull request anyway. The worst that can happen is you'll be politely asked to change something. We love all friendly contributions.
+# Welcome!
 
 We want to ensure a welcoming environment for all of our projects. Our staff follow the [18F Code of Conduct](https://github.com/18F/code-of-conduct/blob/master/code-of-conduct.md) and all contributors should do the same.
```

Comment:
let's add a call to action for the user to familiarize that code of conduct before making a contribution